storage/Utils/Logger.cc

```diff
@@ -58,6 +61,41 @@ namespace storage
     }
 
 
+    class LogfileLogger : public Logger
+    {
+    public:
+
+	virtual void write(LogLevel log_level, const std::string& component, const std::string& file,
+			   int line, const std::string& function, const std::string& content) override;
+
+    };
+
+
+    void
+    LogfileLogger::write(LogLevel log_level, const std::string& component, const std::string& file,
+			 int line, const std::string& function, const std::string& content)
+    {
+	FILE* f = fopen("/var/log/libstorage.log", "ae");
```

Comment:
Yes, the name and access mode are constants. The name is on purpose a constant and the complete class is only for examples and test-cases. So I did not add functions to change the name.